platform/osx/display_server_osx.mm

```diff
@@ -2285,6 +2285,13 @@ static void displays_arrangement_changed(CGDirectDisplayID display_id, CGDisplay
 	return Size2i();
 }
 
+float DisplayServerOSX::screen_get_refresh_rate(int p_screen) const {
+	_THREAD_SAFE_METHOD_
+	// There's no way to get the screen refresh rate using AppKit
```

Comment:
Internal displays are 60Hz, external can have any refresh rate.

API to get refresh rate. https://developer.apple.com/documentation/coregraphics/1454661-cgdisplaymodegetrefreshrate

Enumeration example:
```obj-c++	
	NSArray *screenArray = [NSScreen screens];
	for (int i = 0; i < [screenArray count]; i++) {
		NSDictionary *description = [[screenArray objectAtIndex:i] deviceDescription];
		double rate = CGDisplayModeGetRefreshRate(CGDisplayCopyDisplayMode([[description objectForKey:@"NSScreenNumber"] unsignedIntValue]));
		if (rate == 0) {
			rate = 60.f;
		}
		printf("refresh rate [%d]: %f\n", i, rate);
	}
```

Reply:
Since this was posted, note that internal displays can now be 120 Hz in the new 14" and 16" MacBook Pros.